ee/maintained-apps/inputs/winget/scripts/paint_dot_net_uninstall.ps1

```
@@ -0,0 +1,27 @@
+# Paint.NET's ProductCode changes with every release, and the .exe and .msi
+# variants register different ones. The UpgradeCode is stable across both, so
+# resolve the installed products from it.
+$upgradeCode = '{04A40F40-A207-4B48-AED7-6AA532E43275}'
+$timeoutSeconds = 300
+$successCodes = @(0, 3010, 1641)
+
+try {
+    $inst = New-Object -ComObject "WindowsInstaller.Installer"
+    $productCodes = @()
+    try { $productCodes = @($inst.RelatedProducts($upgradeCode)) } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)